contrib/cirrus/lib.sh

```diff
@@ -345,6 +345,21 @@ install_test_configs(){
 remove_packaged_podman_files(){
     echo "Removing packaged podman files to prevent conflicts with source build and testing."
     req_env_var OS_RELEASE_ID
+
+    # If any binaries are resident they could cause unexpected pollution
+    for unit in io.podman.service io.podman.socket
+    do
+        for state in enabled active
+        do
+            if ! systemctl is-$state $unit
```

Comment:
I have to conclude that the `!` is a typo?

Reply:
I would suggest `-q` (`systemctl -q is-etc`) to avoid cluttering stdout. And of course please remove the `!`.

Reply:
The logs also agree with you :smile: